MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -135,9 +142,7 @@ protected void initialize(@NonNull final Context context, @NonNull final MapboxM
     View view = LayoutInflater.from(context).inflate(R.layout.mapbox_mapview_internal, this);
```

Comment:
Lazy loading would also mean lazy inflating. We would probably need to remove `mapbox_mapview_interal` an only create/add views when they a requested.

Reply:
Good point, adjusted to lazy inflate view components.